meeting_minutes/230420_SARIF_TC_74.md

```diff
@@ -0,0 +1,140 @@
+  * Action is up for adoption
+  * ONGOING
+* ACTION on Jeff to provide an initial seed page at github to foster the future roadmap / extension discussion
+  * CLOSED
+* ACTION on David K. to talk to Jeff to roll over to meeting 2023-APR-20
+  * CLOSED
+
+## 1.7 Identification of SARIF TC voting members (Co-Chair David)
+
+### 1.7.1 Prospective members attending their first meeting
+
+### 1.7.2 Members attaining voting rights at the end of this meeting
+
+* David Malcom
+
+### 1.7.3 Members losing voting rights if they have not joined this meeting by the time it ends
+
+* Michael Fanning
+
+### 1.7.4 Members who previously lost voting rights who are attending this meeting
+
+### 1.7.5 Members who have declared a leave of absence
+
+* Nathan Baird
+
+# 2. Future Meetings
+
+## 2.1 Future meeting schedule (Co-Chair Keaton)
+
+- Proposed Teleconference (Thursday at 08:00 PT / 15:00 UTC for 1.5 hours)
+    ```
+    May 25
+    June 15
+    ```
+- Possible face-to-face meeting when pandemic permits
+
+# 3. Liaisons
+
+## 3.1 [OpenSSF](https://openssf.org/) Security Tools group
+
+> Liaison Group: ForAllSecure/Thanassis Avgerinos, Contrast Security/Jeff Williams, Microsoft/Michael Fanning
+
+* Report status and any communication from OpenSSF Security Tools
+  * No news
+* Compose a liaison statement to OpenSSF Security Tools if appropriate
+  * No news
+
+# 4. Discussion
+
+## 4.1 Review status on finalizing SARIF 2.1 errata [List of next steps - github issue #509](https://github.com/oasis-tcs/sarif-spec/issues/509)
+
+* David K.: Microsoft supported with funding a person to assist the TC in editing who is already collaborating with the chair to progress the errata
+
+## 4.2 Review current state of ecosystem ongoing work
+
+* No news
+
+## 4.3 Review outcomes of subgroup discussions
+
+* No news
+
+## 4.4 Discuss end-to-end results management
+
+* No news
+
+## 4.5 List of small, non-breaking open requests to consider as a dot release of the SARIF standard
+
+* No news
+
+## 4.6 Review Roadmap
+
+* Draft provided by Jeff will be discussed in later meetings
+
+## 4.7 Discuss SARIF's relationship to other relevant standards
+
+* No news
+
+# 5. Other Business
+
+* David K.: Thanassis had an outstanding request for SARIF to support fuzzing and put it to use with his fuzzing framework and sees no need for changes to the format.
+* Pavel: What is the status of supporting dynamic (DAST) and interactive (IAST) tools?
+* David K.: Walks all through the relevant section in the document provided by Jeff per [the pull request "Create Future.md" #560](https://github.com/oasis-tcs/sarif-spec/pull/560)
+
+# 6. Resolutions and Decisions reached (by 10 minutes prior to scheduled meeting end)
+
+## 6.1 End debate of other issues by 10 minutes prior to scheduled meeting end and follow the agenda from this point (Co-Chair David)
+
+## 6.2 Review of Decisions Reached (Secretary Stefan)
+
+* DECISION to meet next on May 25, 2023
```

Comment:
Wasn't there also the decision to meet on June 15?

Reply:
Good catch.

Reply:
I suggest to leave as is. We often do mention in decisions only the next meeting. During the meeting I edit the proposed meetings in the minutes to match the outcome of the discussion. Now we already note May 25 three times: In the proposed section, in the decisions, and in the next meeting section.